appconfig/passwords/accesslist.txt

```diff
@@ -21,4 +21,6 @@ dqe:pass
 dqedb:pass
 segmentedtickerplant:pass
 filealerter:pass
+tailer:pass
+torquser:pass
```

Comment:
What does torquser do, seems it wasn't in previous accesslist.txt files?

Reply:
I get a failed: access error without it

Reply:
looks like you're missing an `appconfig/passwords/tailer.txt` - this would be what the tailer would load to set it's username & password that it'll use to connect to other procs. `torquser:pass` is the default if this file isn't found

Reply:
should that be in the FSP or normal torq?

Reply:
good point, should actually be in TorQ, in `config/` (as opposed to `appconfig/` like I said in previous comment)

Reply:
okay, I have added that file to my other PR, should I now remove torquser from the access file?

Reply:
yep